StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/Link/PaymentDetails.swift

```diff
@@ -20,17 +20,231 @@ typealias ConsumerSessionWithPaymentDetails = (session: ConsumerSession, payment
  */
 final class ConsumerPaymentDetails: Decodable {
     let stripeID: String
+    let details: Details
+    var isDefault: Bool
 
-    init(stripeID: String) {
+    // TODO(csabol) : Billing address
```

Comment:
maybe change to TODO(link), or just delete if we don't plan on adding it? (also, if it causes issues with merging, then ignore my comment)